src/Particular.CodeRules/tools/install.ps1

```diff
@@ -0,0 +1,6 @@
+﻿param($installPath, $toolsPath, $package, $project)
```

Comment:
@danielmarbach @SimonCropp there's a special convention for installing analyzers with project.json, but for the older format (packages.config) they have to be installed via a powershell script only: https://docs.nuget.org/create/analyzers-conventions

(this is because the Analyzers node in the project is not a real msbuild construct, but something Visual Studio project model creates at runtime)